doc/autodocs/doxygen.conf

```diff
@@ -13,10 +13,12 @@ TAB_SIZE = 8
 
 INPUT = ../../storage
 FILE_PATTERNS = Actiongraph.h Devicegraph.h Environment.h FreeInfo.h Graphviz.h	\
-	SimpleEtcFstab.h SimpleEtcCrypttab.h Storage.h UsedFeatures.h Version.h
+	SimpleEtcFstab.h SimpleEtcCrypttab.h Storage.h UsedFeatures.h Version.h	\
+	CompoundAction.h
```

Comment:
CompoundAction.h does not include any documentation.

Reply:
There is a new static method with `@throw DeviceNotFound` doc, so we need to include it in doxygen to generate the catches file for bindings.